src/MusicStore/config.json

```diff
@@ -8,7 +9,8 @@
         "DefaultConnection": {
             // Use a shared (and running) LocalDB database when executing in IIS e.g.
```

Comment:
Consider moving comments outside of the config.json file. This is a bit of a mix of concerns: config.json should only provide the configuration required, not the logic or annotations for the logic. Additionally, if in the future we change the JSON parser it might choke on comments, as they aren't part of [the spec](http://www.ietf.org/rfc/rfc7159.txt).